web/src/routes/news/containers/LocalNewsDetailsPage.js

```diff
@@ -35,11 +35,12 @@ export const LocalNewsDetailsPage = ({
     const error = new NotFoundError({ type: LOCAL_NEWS_TYPE, id, city, language })
     return <FailureSwitcher error={error} />
   }
+  const linkedContent=replaceLinks(localNewsElement.message)
```

Comment:
Take care of correct formatting.
```suggestion
  const linkedContent = replaceLinks(localNewsElement.message)
```
Either use auto formatting (possibly `STRG` + `ALT` + `L`?) or fix errors showing up in the IDE or after running `yarn lint` manually.

Reply:
Changed the regex, thx!